Form-Controls/index.html

```diff
@@ -1,28 +1,87 @@
-  <meta name="description" content="">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="stylesheet" href="styles.css">
-</head>
-<body>
-  <header>
-    <h1>Product Pick</h1>
-  </header>
-  <main>
-  <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
-  </form>
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>My form exercise</title>
+    <meta name="description" content="" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <header>
+      <h1>Product Pick</h1>
+    </header>
 
-  </main>
-  <footer>
-    <!-- change to your name-->
-    <h2>By HOMEWORK SOLUTION</h2>
-  </footer>
-</body>
+    <main>
+      <form class="container">
+        <!--  Name and Last Name -->
 
-</html>
\ No newline at end of file
+        <div class="flex-row1">
+          <div class="couple-column">
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="name" required />
+          </div>
+          <div class="couple-column">
+            <label for="lastname">Lastname:</label>
+            <input type="text" id="lastname" name="lastname" required />
+          </div>
+        </div>
+
+        <!-- T-Shirt Color-Size -->
+        <div class="flex-row2">
+          <div class="couple-column">
+            <label for="color">T-Shirt Color:</label>
+            <select id="color" name="color" required>
+              <option value="" disabled selected>Select</option>
+              <option value="red">Red</option>
+              <option value="blue">Blue</option>
+              <option value="green">Green</option>
+            </select>
+          </div>
+          <div class="couple-column">
+            <label for="size">T-Shirt Size:</label>
+            <select id="size" name="size" required>
+              <option value="" disabled selected>Select</option>
+              <option value="XS">XS</option>
+              <option value="S">S</option>
+              <option value="M">M</option>
+              <option value="L">L</option>
+              <option value="XL">XL</option>
+              <option value="XXL">XXL</option>
+            </select>
+          </div>
+        </div>
+
+        <!-- Email and Delivery Date -->
+        <div class="flex-row3">
+          <div class="couple-column">
+            <label for="email">Email:</label>
+            <input type="email" id="email" name="email" required />
+          </div>
+          <div class="couple-column">
+            <label for="deliveryDate"
+              >Delivery Date
+              <span class="small-text">(within the next 4 weeks</span>):</label
+            >
+            <input
+              type="date"
+              id="deliveryDate"
+              name="deliveryDate"
+              required
```

Comment:
Great work on the HTML element restrictions. On the date min and max, what could you change so that it dynamic changes to min=Today and max=Today+4 weeks?
This is a little trickier and requires JS or PHP.